main/guides/zoe/contract-requirements.md

```diff
@@ -46,6 +46,13 @@ to pass in any arguments that should not be part of the public terms.
 `privateArgs` is an object with keys and values as decided by the caller of
 `startInstance`. If no private arguments are passed, `privateArgs` is undefined.
 ```js
+const start = (zcf, privt Facet](/reference/zoe-api/zoe-contract-facet.md) and is
```

Comment:
This does not look like valid markdown syntax. Looking at https://1750ad84.documentation-7tp.pages.dev/guides/zoe/contract-requirements.html it does not seem to render as you intend.

Reply:
Following the preview link from the `Checks` tab of the PR, we can see this code snippet at https://1750ad84.documentation-7tp.pages.dev/guides/zoe/contract-requirements.html
<img width="838" alt="Screenshot 2023-02-18 at 9 09 44 AM" src="https://user-images.githubusercontent.com/1002798/219870350-71e69bfd-fd93-42e2-a7c0-37a61cb0b866.png">

Reply:
Resolving all conversations that were only about the fubar contract-requirements.md